Microsoft.Dynamics365.UIAutomation.Sample/TestSettings.cs

```diff
@@ -36,6 +37,22 @@ public static class TestSettings
             DriversPath = Path.IsPathRooted(DriversPath) ? DriversPath : Path.Combine(Directory.GetCurrentDirectory(), DriversPath) 
         };
 
+        // Create a new options instance, copy of the share, to use just in the current test, modifications in test will not affect other tests
+        public static BrowserOptions Options => new BrowserOptions 
+        {
```

Comment:
Hi @bwmodular, Hi @TYLEROL, Hi @dtu11 

I have a concern here, I move the previous Options to SharedOptions, because this is a Static Property and a change in one Tests will affect the follows tests (because the Tests are not ordered, that cannot be the desired effect). By getting a new instance each time, the Tester is able to set the option value just for a particular test. That works fine. I already check that. However I don't work with **.runsettings** as @bwmodular  mentions in [this comment](https://github.com/microsoft/EasyRepro/issues/738#issuecomment-580147055). **May this change broke some configuration in DevOps from somebody that use ‘.runsettings’?** Can be this ".runsettings" be mapped somehow direct to the Public Instance of TestSettings, instead truth the app.config?
```
<!-- Parameters used by tests at run time -->
  <TestRunParameters>
    <Parameter name="OnlineUsername" value="http://..." />
    <Parameter name="OnlinePassword" value="Admin" />
    <Parameter name="OnlineCrmUrl" value="Password" />
  </TestRunParameters>
```
In such case, that will be ignored. ShareOptions should be used instead, except if such mapping happen previous each tests run. 
That can be also a problem if somebody set Options or SharedOptions in [ClassInitialize] or [TestInitialize] methods.
I provide safe mechanism in SetOptions method. See please the demo samples that I add in the last commits. These should help also the “Testers that aren't developers” that @dtu11 mentions yesterday. Maybe we should add this to the Wiki.

Thanks